packages/upgrade/test/upgrade.test.ts

```diff
@@ -35,7 +35,7 @@ describe('Test upgrade', async function () {
   });
 
   test('should output the correct data from upgrade', () => {
-    const report = (reporter as any).report as Report;
+    const { report } = reporter;
```

Comment:
Please keep the report private, you can use `/* eslint-disable ... */` here

Reply:
see prev. comment. doing the type cast is worse. why does it matter if the report is public?